### PR TITLE
Update Velocity Control

### PR DESCRIPTION
**Pull request by AldoArriagaOrta**
Merge of current branch with velocity control for LZ

### CODE REVIEW

File: include/dji_osdk_ros/vehicle_wrapper.h [hunk 1]
```diff
@@ -123,6 +123,24 @@ namespace dji_osdk_ros
       bool monitoredLanding(ACK::ErrorCode& ack, int timeout);
       bool startForceLanding(int timeout);
       bool moveByPositionOffset(ACK::ErrorCode& ack, int timeout, MoveOffset& p_offset);
+      bool monitoredTakeoff(ACK::ErrorCode& ack, int timeout);
```
Comment (ciaran-helgen) [inline include/dji_osdk_ros/vehicle_wrapper.h]: Duplicate of line 122

File: include/dji_osdk_ros/vehicle_wrapper.h [hunk 2]
```diff
@@ -123,6 +123,24 @@ namespace dji_osdk_ros
...
+      bool monitoredLanding(ACK::ErrorCode& ack, int timeout);
```
Comment (ciaran-helgen) [inline include/dji_osdk_ros/vehicle_wrapper.h]: Duplicate of line 123

File: include/dji_osdk_ros/vehicle_wrapper.h [hunk 3]
```diff
@@ -123,6 +123,24 @@ namespace dji_osdk_ros
...
+      bool startForceLanding(int timeout);
```
Comment (ciaran-helgen) [inline include/dji_osdk_ros/vehicle_wrapper.h:125]: Duplicate of line 124

File: include/dji_osdk_ros/vehicle_wrapper.h [hunk 4]
```diff
@@ -123,6 +123,24 @@ namespace dji_osdk_ros
...
+      bool startConfirmLanding(int timeout);
+      bool cancelLanding(int timeout);
+      bool moveByPositionOffset(ACK::ErrorCode& ack, int timeout, MoveOffset& p_offset);
```
Comment (ciaran-helgen) [inline include/dji_osdk_ros/vehicle_wrapper.h]: Duplicate of line 125

**Review by ciaran-helgen — CHANGES_REQUESTED** [review]
Remove duplicate lines 126 to 128, and 131

File: include/dji_osdk_ros/vehicle_wrapper.h (hunk 4 above)
Comment (AldoArriagaOrta) [reply include/dji_osdk_ros/vehicle_wrapper.h]: Updated

File: include/dji_osdk_ros/vehicle_wrapper.h (hunk 2 above)
Comment (AldoArriagaOrta) [reply include/dji_osdk_ros/vehicle_wrapper.h]: Updated

File: include/dji_osdk_ros/vehicle_wrapper.h (hunk 1 above)
Comment (AldoArriagaOrta) [reply include/dji_osdk_ros/vehicle_wrapper.h]: Updated. This one would have caused trouble as the definition is really different

**Pull request merged**